tests/Integration/ApiPlatform/EndPoint/ApiTestCase.php

```diff
@@ -65,6 +65,10 @@ protected static function createClient(array $kernelOptions = [], array $default
             $defaultOptions['headers']['accept'] = ['application/json'];
         }
 
+        if (!isset($defaultOptions['headers']['content-type'])) {
```

Comment:
Good call adding this as the default content type, but the tests below still need to define it I guess it's because of the patch thing right?
So I think this should be the main thing we can try and improve (I mean by that trying not to have to specify this very specific header since there's no alternative available, see my review description for more details)

Reply:
Yes, there is just the testProtectedEndpoints test which still needs to be specified, because of the generic nature of the test